src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  max-width: 1200px;
```

Comment:
Header should be full screen width for any screens. For now it looks like this
![2023-05-17_16h37_24](https://github.com/mate-academy/layout_moyo-header/assets/61904995/725a2759-6d1d-46cc-8b4b-6b625ef541b4)


```suggestion

```